src/NzbDrone.Core/Notifications/Webhook/WebhookMovieFile.cs

```diff
@@ -27,6 +28,7 @@ public WebhookMovieFile(MovieFile movieFile)
         public int QualityVersion { get; set; }
         public string ReleaseGroup { get; set; }
         public string SceneName { get; set; }
+        public string IndexerFlags { get; set; }
```

Comment:
Again the only place we fire include `IndexerFlags` is `OnGrab` which would come from `WebhookRemoteMovie` and not `WebhookMovieFile`, is this intended for Grab notifications or Import notifications?